nextcord/guild.py

```diff
@@ -3079,6 +3080,41 @@ async def ban(
 
         await self._state.http.ban(user.id, self.id, delete_message_seconds, reason=reason)
 
+    async def bulk_ban(self, users: List[Snowflake], *, delete_message_seconds: int = 0) -> BulkBan:
+        """|coro|
+
+        Bans users from a guild in bulk. This has a limit of 200 users
+
+        The list of users must meet the :class:`abc.Snowflake` abc.
+
+        You must have the :attr:`~Permissions.ban_members` and :attr:`~Permissions.manage_guild` permissions to
+        do this.
+
+        Parameters
+        ----------
+        users: :class:`abc.Snowflake`
+            The users to ban from a guild.
+        delete_message_seconds: :class:`int`
+            The number of seconds worth of messages to delete from these users.
+            Can range from 0 to 604800 seconds (7 days).
+
+            Defaults to 0.
+
```

Comment:
The errors that are thrown from this function should be documented here. `Forbidden` should be thrown if there aren't sufficient permissions, and `HTTPException` should be thrown if no users were banned.